src/main/kotlin/no/nav/familie/ef/mottak/service/MappeService.kt

```diff
@@ -0,0 +1,111 @@
+package no.nav.familie.ef.mottak.service
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.ef.mottak.integration.IntegrasjonerClient
+import no.nav.familie.ef.mottak.repository.domain.Søknad
+import no.nav.familie.ef.mottak.service.MappeSøkestreng.OPPLÆRING
+import no.nav.familie.ef.mottak.service.MappeSøkestreng.SELVSTENDIG
+import no.nav.familie.ef.mottak.service.MappeSøkestreng.SÆRLIG_TILSYNSKREVENDE
+import no.nav.familie.ef.mottak.service.MappeSøkestreng.UPLASSERT
+import no.nav.familie.ef.mottak.util.dokumenttypeTilStønadType
+import no.nav.familie.kontrakter.ef.søknad.Aktivitet
+import no.nav.familie.kontrakter.ef.søknad.SøknadBarnetilsyn
+import no.nav.familie.kontrakter.ef.søknad.SøknadOvergangsstønad
+import no.nav.familie.kontrakter.ef.søknad.Søknadsfelt
+import no.nav.familie.kontrakter.felles.ef.StønadType
+import no.nav.familie.kontrakter.felles.objectMapper
+import no.nav.familie.kontrakter.felles.oppgave.FinnMappeRequest
+import no.nav.familie.kontrakter.felles.oppgave.FinnMappeResponseDto
+import no.nav.familie.kontrakter.felles.oppgave.MappeDto
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class MappeService(
+    private val integrasjonerClient: IntegrasjonerClient,
+    private val søknadService: SøknadService
+) {
+
+    val log: Logger = LoggerFactory.getLogger(this::class.java)
```

Comment:
Det går å bruke `javaClass` i stedet for `this::class.java` :) 
```suggestion
    private val logger = LoggerFactory.getLogger(javaClass)
```